.formatter.exs

```diff
@@ -0,0 +1,4 @@
+# Used by "mix format"
+[
+  inputs: ["{mix,.formatter}.exs", "{config,lib,test}/**/*.{ex,exs}"]
```

Comment:
We can drop this and add it to the elixir branch if need be

Reply:
Just did :+1: